### DISCUSSION TITLE
password traffic from login page to opanam-server - possibility of password hash ?

### DISCUSSION THREAD

**Discussion by dei01707** [OpenIdentityPlatform/OpenIG#66] [Q&A]
Hello community,

is there a possibility to hash password traffic from loginpage to openam server?
We use OpenAM 14.8.2
In our web-application-firewall-log we can see, the user in cleartext 🙁
Hope you can help me.

Regards
Claudia

**Comment by vharseko** [comment]
password is stored in hashed form
If you hash it during verification, you will have to store it in plaintext.
use ssl
